quarkus/runtime/src/main/java/org/keycloak/quarkus/runtime/KeycloakMain.java

```diff
@@ -80,7 +79,7 @@ public static void start(List<String> cliArgs, PrintWriter errorWriter) {
      */
     @Override
     public int run(String... args) throws Exception {
-        if (isDevMode()) {
+        if (getProfile().equals("dev")) {
```

Comment:
Wouldn't make more sense to have this as a method on the `Environment` class similar to others?